packages/nestjs/package.json

```diff
@@ -41,23 +43,23 @@
         "/lib"
     ],
     "devDependencies": {
+        "@backtrace/node": "^0.3.1",
```

Comment:
why backtrace/node is a dev dependency here?

Reply:
good catch, should be in normal deps